app/src/main/java/com/sunasterisk/englishflashcard/ui/BaseFragment.kt

```diff
@@ -0,0 +1,19 @@
+package com.sunasterisk.englishflashcard.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+abstract class BaseFragment : Fragment() {
+    protected abstract val layoutResource: Int
```

Comment:
viết như này nó sẽ check id là layout 
```suggestion
    @get:layoutRes
    protected abstract val layoutResource: Int
```